### ISSUE TITLE
Latest version of Pillow breaks current implementation in html_generator.py.

### ISSUE THREAD

**Issue by jllllll** [oobabooga/text-generation-webui#2958]
### Describe the bug

Pillow 10.0.0 removed `ANTIALIAS` from `PIL.Image`. Current implementation requires 9.5.0, however the requirements.txt currently allows for 10.0.0 to be installed.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

Add new characters with png images and load the webui in chat mode.

### Logs

```shell
Traceback (most recent call last):
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\installer_files\env\lib\site-packages\gradio\routes.py", line 427, in run_predict
    output = await app.get_blocks().process_api(
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\installer_files\env\lib\site-packages\gradio\blocks.py", line 1323, in process_api
    result = await self.call_function(
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\installer_files\env\lib\site-packages\gradio\blocks.py", line 1051, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\installer_files\env\lib\site-packages\anyio\to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\installer_files\env\lib\site-packages\anyio\_backends\_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\installer_files\env\lib\site-packages\anyio\_backends\_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\text-generation-webui\extensions\gallery\script.py", line 71, in generate_html
    image_html = f'<img src="file/{get_image_cache(path)}">'
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\text-generation-webui\modules\html_generator.py", line 144, in get_image_cache
    img = make_thumbnail(Image.open(path))
  File "G:\F\Projects\AI\text-generation-webui\one-click-installers-test\text-generation-webui\modules\html_generator.py", line 132, in make_thumbnail
    image = ImageOps.fit(image, (350, 470), Image.ANTIALIAS)
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
```

### System Info

```shell
Windows 10
GPU: GTX 1080ti
```

**Comment by CuddleBear92** [comment]
This also happens on startup for me with the same error, editing the requirements.txt file to say `Pillow==9.5.0` and running the update fixed it again for me.

**Comment by missionfloyd** [comment]
https://pillow.readthedocs.io/en/stable/deprecations.html#constants

**Comment by Macjja** [comment]
It appears to work if you replace the ANTIALIAS with its replacement, Resampling.LANCZOS
I tested this in the Colab

**Comment by AniketShinde1998** [comment]
How to fix this error is this problem arising due to version compatibility of easyocr, python, pillow or what because i tried lot of things it is not working will pillow == 9.5.0 specifically fix this error? 


this is the code:
import easyocr
import cv2
img = cv2.imread("../data_files/temp_files/captcha.png", cv2.IMREAD_GRAYSCALE)
reader = easyocr.Reader(['en'], gpu=False)
result = reader.readtext(img, detail=0)
print(result[0].replace(" ", "").upper())

this is the error:
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'

**Comment by AniketShinde1998** [comment]
> It appears to work if you replace the ANTIALIAS with its replacement, Resampling.LANCZOS I tested this in the Colab
> 
> EDIT: It appears someone has already opened a pull request for this fix, #2954

how to do that where do we replace that?

**Comment by jllllll** [comment]
@AniketShinde1998 You can see what was changed here to fix this in `modules/html_generator.py`: https://github.com/oobabooga/text-generation-webui/pull/2954/files

You can install pillow 9.5.0 as well, but `ANTIALIAS` was deprecated anyway and `LANCZOS` is what should be used.
